bootstrap/README.md

```diff
@@ -268,13 +268,45 @@ $ cargo run --example upload -- \
 
 Prepare upgrade arguments
 ```shell
+# https://internetcomputer.org/docs/references/bitcoin-how-it-works#api-fees-and-pricing
+$ TESTNET_FEES="record { 
+  get_utxos_base = 20_000_000 : nat;
+  get_utxos_cycles_per_ten_instructions = 10 : nat;
+  get_utxos_maximum = 4_000_000_000 : nat;
```

Comment:
This also called `minimum` in the reference doc and should be updated?

Reply:
`*_maximum` is a maximum amount of cycles that can be charged, which in doc translates to 'Minimum cycles to send with call' that makes more sense from a developer point of view. So I'll leave it as is.